packages/react/src/combobox/ComboboxBase/ComboboxList.tsx

```diff
@@ -1,35 +1,12 @@
-import { forwardRef, PropsWithChildren } from 'react';
+import { forwardRef, HTMLAttributes } from 'react';
 import { Box } from '../../box';
-import { FieldMaxWidth, tokens } from '../../core';
 
-export type ComboboxListProps = PropsWithChildren<{
-	block: boolean;
-	isOpen: boolean;
-	maxWidth: FieldMaxWidth;
-}>;
+export type ComboboxListProps = Omit<HTMLAttributes<HTMLUListElement>, 'color'>;
 
 export const ComboboxList = forwardRef<HTMLUListElement, ComboboxListProps>(
-	function ComboboxList({ children, isOpen, block, maxWidth, ...props }, ref) {
+	function ComboboxList({ children, ...props }, ref) {
```

Comment:
You've modified ComboboxList, but it appears you've removed it's usage from the above file (ComboboxBase). Is that right?